packages/core-components/src/lib/unsorted/ui/Modal.svelte

```diff
@@ -0,0 +1,62 @@
+<script context="module">
+	export const evidenceInclude = true;
+</script>
+
+<script>
+	import CloseIcon from '../icons/CloseIcon.svelte';
+	export let open = false;
+	export let title = '';
+	export let buttonText = '';
+	export let innerText = '';
+	const isOpen = () => {
+		open = !open;
+	};
+</script>
+
+<button
+	on:click={() => isOpen()}
+	class="font-ui font-normal text-base no-underline border my-2 rounded-lg py-2 px-3 inline-block transition hover:ease-in hover:border-blue-200 hover:bg-blue-100 hover:no-underline hover:text-gray-800 focus:text-gray-900 focus:border-blue-200"
+	>{buttonText}</button
+>
+{#if open}
+	<!-- svelte-ignore a11y-click-events-have-key-events -->
```

Comment:
I'm not super thrilled about having these ignored; for the accordion I think it's okay. Can we add a listener for the `escape` key to close the modal if it is open? That should resolve the issue and add a behavior that users will probably expect.

We also may want to include support for clickOutside, potentially as a prop to toggle it (default off), there is an action that can be used in the `core-components/src/clickOutside.js` file

Reply:
Svelte-ignore removed by making this a button rather than a div